integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -101,6 +101,13 @@
                             aid: '1234' //Use 1234 for test ads
                         }
                     },
+		    {
+   			bidder: 'advenue',
```

Comment:
Since we have params in md file, no need to update this page. In future we will remove this page from repo

Reply:
Deleted package-lock.json and pbjs_example_gpt.html from master